packages/grid/src/elements/Col.tsx

```diff
@@ -19,80 +19,79 @@ import { StyledCol } from '../styled';
 import useGridContext from '../utils/useGridContext';
 
 export interface IColProps extends HTMLAttributes<HTMLDivElement> {
-  /**
-   * Determine the size, relative to the total number of `columns` in the grid,
-   * for all screen sizes
-   */
+  /** Sets the total number of grid `columns` that the col spans on all screen sizes */
   size?: GRID_NUMBER;
-  /** Determine the size for extra-small screen sizes */
+  /** Defines the col size on extra-small screens.
+   *  See [breakpoints](https://garden.zendesk.com/components/theme-object#default_theme).
```

Comment:
Let's use the `website`-local URL here, and maybe change to `/components/theme-object#breakpoints`